ndb/src/google/cloud/ndb/model.py

```diff
@@ -80,6 +83,7 @@
 ]
 
 
+_EMPTY_LIST_MEANING = 24
```

Comment:
Unused? Fairly cryptic.

Reply:
Yeah this was part of my deleted implemenation. Sorry.